images/stackstorm/bin/entrypoint-1ppc.sh

```diff
@@ -5,24 +5,30 @@ crudini --set ${ST2_CONF} auth    api_url     ${ST2_API_URL}
 crudini --set ${ST2_CONF} mistral api_url     ${ST2_API_URL}
 crudini --set ${ST2_CONF} mistral v2_base_url ${ST2_MISTRAL_API_URL}
 
+# Configure CORS to accept any source
+# st2api gunicorn process is directly exposed to clients in 1ppc mode
 crudini --set ${ST2_CONF} api allow_origin '*'
 
+# Overwrite nginx config for st2web
+# to support load balancing to st2api, st2auth and st2stream
+mv /etc/nginx/conf.d/st2-1ppc.conf.disable /etc/nginx/conf.d/st2.conf
```

Comment:
Why move file here at runtime? Why not copy directly to this location in the Dockerfile?

Reply:
I've fixed this up so `st2-1ppc.conf` is written into the image. The appropriate entrypoint script creates a symlink from `st2.conf` -> `st2-base.conf` or `st2-1ppc.conf` as required.